addons/bestja_detailed_reports/views/detailed_reports_summary.xml

```diff
@@ -65,6 +67,21 @@
                 ('detailed_report.state', '!=', 'draft'),
             ]</field>
             <field name="context">{
+                'search_default_responsible_project': active_id,
+                'search_default_accepted': True,
+            }</field>
+        </record>
+
+        <record model="ir.actions.act_window" id="action_detailed_reports_product_summary_lvl0">
```

Comment:
This record is exactly the same as the one with the id `action_detailed_reports_product_summary_lvl1`. The `_compute_responsible_project` method returns from the same branch of code for values `0` and `1`. Menu entries are analogous for values `0` and `1`. What's the purpose of splitting this into two?